app/views/providers/confirm_offices/show.html.erb

```diff
@@ -19,15 +19,10 @@
     <%= govuk_form_group(
           show_error_if: @error,
         ) do %>
-      <%
-        options = [
-          { value: :yes, label: t('generic.yes') },
-          { value: :no,  label: t('.no_another_office') }
-        ]
-      %>
+
       <%= form.govuk_collection_radio_buttons(
             :correct,
-            options,
+            form.yes_no_radio_button_array,
```

Comment:
Won't doing this lose the translation for no ('No, another office is handling this application') and just replace it with 'No'? Shouldn't we keep the `options` block?

Reply:
D'oh! Good catch, I'll revert this change